_episodes/08-command-substitution.md

```diff
@@ -237,14 +235,13 @@ loops.
 > ~~~
 > but for *each* of the `.pdb`-files.
 <!-- solution: for file in *.pdb; do sort $file > $(basename $file .pdb).sorted; done -->
+{: .challenge}
 
 ## Closing remarks
```

Comment:
This chapter is missing "Key Points", so the respective box is empty :)

Reply:
Done